src/lib/radio/_radio-theme.scss

```diff
@@ -7,13 +7,12 @@
     border-color: mat-color($palette);
   }
 
-  .mat-radio-inner-circle {
+  .mat-radio-inner-circle,
+  .mat-radio-ripple .mat-ripple-element:not(.mat-radio-persistent-ripple),
+  &.mat-radio-checked .mat-radio-persistent-ripple,
+  &:active .mat-radio-persistent-ripple {
```

Comment:
A comment with the different states would be probably nice here, but not required 😄